drivers/include/net/netdev.h

```diff
@@ -54,6 +54,7 @@ extern "C" {
 #include <sys/uio.h>
 
 #include "net/netstats.h"
+#include "net/l2filter.h"
```

Comment:
> If you are not happy with that, please provide a follow-up for these cases and let's discuss it there.

This is not how we **should** review our PRs. Rather than trying to get them in quickly, but sloppy, and rely on follow-up PRs, we should aim for our best in the first iteration.
Does it hurt so much to **not** copy a previously applied style that might be flawed? It's just two lines more for less problems .. Since when are unused include files not considered as bad coding? (;